internal/config/config.go

```diff
@@ -246,7 +256,8 @@ func Default() *Config {
 				LibvirtURI:            "",
 				LibvirtContainerImage: versions.LibvirtImage,
 				Measurements:          copyPCRMap(qemuPCRs),
-				EnforcedMeasurements:  []uint32{11, 12},
+				EnforcedMeasurements:  []uint32{4, 8, 9, 11, 12, 13, 15},
+				NVRAM:                 "production",
```

Comment:
Not sure what our policies are for the main branch, but since we don't currently ship production NVRAM I feel like this should be set to `testing` by default.

Reply:
Every release CLI will have to set this to production then. But I guess this can be automated.. So I'm fine with that :+1: